pkg/controller/common/container/container.go

```diff
@@ -24,6 +24,8 @@ const (
 	ElasticsearchImage    Image = "elasticsearch/elasticsearch"
 	KibanaImage           Image = "kibana/kibana"
 	EnterpriseSearchImage Image = "enterprise-search/enterprise-search"
+	FilebeatImage         Image = "beats/filebeat"
+	MetricbeatImage       Image = "beats/metricbeat"
```

Comment:
I was thinking we would have a generic

```go
func BeatImage(beatType beatv1beta1.Type) string {
    return fmt.Sprintf("beats/%s", beatType)
}
```

to be compatible with any official beat image directly.

Reply:
I'd rather keep all images we use defined in a single place and be explicit. And referencing beat from common seems off to me. WDYT?

Reply:
Relates https://github.com/elastic/cloud-on-k8s/pull/3041#discussion_r435826407.
Let's list all the official Beats here.

Reply:
Discussed on Slack: it's about what we "support". If the user is allowed to specify `type: packetbeat` they may feel like we officially support it, which isn't true.

Reply:
+1 to supporting the images of all official Beats and not having to manually spec the Docker image for those that don't have explicit support.

Reply:
We could at least support those that work on k8s like auditbeat and heartbeat.

Reply:
I'd like to address it with a separate PR. Current Heartbeat e2e test is more about testing the "otherbeat" deployed than the Heartbeat itself.